flow-typed/npm/acorn-jsx_vx.x.x.js

```diff
@@ -0,0 +1,45 @@
+// flow-typed signature: 7dde706976fd7214173b1fa27157eacb
```

Comment:
@danpersa Unfortunately in order to check type-integrity we need the library type definitions provided by flow-typed.

But flow-typed install does not work on travis because of how flow-typed accesses the filesystem. Furthermore there are other reasons to include the type definitions in the repository. See here: https://github.com/flowtype/flow-typed/issues/413#issuecomment-257122003.